tests/devices/default_qubit/test_default_qubit_tracking.py

```diff
@@ -255,27 +255,6 @@ def test_single_expval(mps, expected_exec, expected_shots):
         assert dev.tracker.totals["shots"] == 3 * expected_shots
 
 
-@pytest.mark.usefixtures("use_new_opmath")
-@pytest.mark.xfail(reason="bug in grouping for tracker with new opmath")
-def test_multiple_expval_with_prods():
-    """Can be combined with test below once the bug is fixed - there shouldn't
-    be a difference in behaviour between old and new opmath here"""
-    mps, expected_exec, expected_shots = (
-        [qml.expval(qml.PauliX(0)), qml.expval(qml.PauliX(0) @ qml.PauliY(1))],
-        1,
-        10,
-    )
-    dev = qml.device("default.qubit")
-    tape = qml.tape.QuantumScript([], mps, shots=10)
-
-    with dev.tracker:
-        dev.execute(tape)
-
-    assert dev.tracker.totals["executions"] == expected_exec
-    assert dev.tracker.totals["simulations"] == 1
-    assert dev.tracker.totals["shots"] == expected_shots
-
-
 @pytest.mark.usefixtures("use_legacy_opmath")
 def test_multiple_expval_with_tensors_legacy_opmath():
```

Comment:
```suggestion
def test_multiple_expval_with_prod():
```
This test can just run as normal now, it was only separated out as only for legacy opmath because this bug caused it to fail for the new opmath